finance/portfolio-performance/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        buchen portfolio 0.49.4
+name                portfolio-performance
+categories          finance
+platforms           darwin
+license             EPL-1
+maintainers         @janosch mailbox.org:janosch1 openmaintainer
+description         A simple tool to calculate the overall performance of an investment portfolio.
+long_description    An open source tool to calculate the overall performance of an investment portfolio \
+                    - across all accounts - using True-Time Weighted Return or Internal Rate of Return.
+homepage            https://www.portfolio-performance.info/en/
+
+depends_build       port:maven3 \
+                    bin:java:openjdk15
```

Comment:
I'm presuming that since this is a Java project, Java would be required at both build time and runtime. I'm also going to assume that more than just the Java binary is required.

I would recommend using the Java PortGroup to set this all up for you. If you do so, this should be changed to a `depends_build-append`.

Reply:
depends_build is definitly wrong for JDK, it should be depends_lib. I'll change that.

The java PG is currently broken for Big Sur. See https://trac.macports.org/ticket/61445. I would prefer not using it until this issue is resolved.

Reply:
@harens Are you okay with not using the java PG for the reason state above? If not, I will change the port to use the java PG. 

I'd like to get this PR ready to merge.

Reply:
Apologies for the delay. Completely understand that you want to get this merged as soon as possible.

Other people might have different opinions on this, but assuming everything works at runtime, I think what you suggest is fine for the time being.

Having said that, once the issue with the Java PG is fixed, I'd prefer if we revert this. What happens if the user has openjdk11 installed? The [relevant README section](https://github.com/buchen/portfolio#building-with-maven) says this if fine, but the portfile requires openjdk15. What if they the have Oracle JDK? This is why there's a PortGroup to simplify this process.

If we stick with your suggestion for now, I'd prefer openjdk11 simply because it's an LTS release, unless there's any particular reason for going with openjdk15.

On a quick side-note, until we can use the Java PG, do you think it's worth setting `JAVA_HOME` in the scenario where the user has it pointing to a different SDK to the one in the portfile?

Thanks again for writing this portfile. A lot of great progress has been made so far.

_(Just reading through the trac ticket, I noticed you sent a PR with your suggestion for a workaround https://github.com/macports/macports-ports/pull/9617. Thank you for your work on this 👍.)_

Reply:
@harens I changed my mind regarding the java PG. As I have maven3 as dependency (which in turn uses the java PG), I will run into the java PG bug anyway so it does not really make sense to bypass it in the port of portfolio-performance. So with the latest push I added the java PG and changed `depends_build` to `depends_build-append` as requested.